app/models/schedule_rule.rb

```diff
@@ -67,6 +67,8 @@ def weeks_display
     end
   end
 
+  @hours = %w(7 8 9 10 11 12 13 14 15 16 17 18)
```

Comment:
I don't think these are actually used?

Reply:
In lines 758 and 1330 of scripts.coffee:
```
for time in scheduleTimes
  selected = if time == script.schedule_time then 'selected' else null
  formatted = time
  if time in constants.hours # Don't format none, morning, evening
  pmOrAm = if time >= 12 then ' PM' else ' AM'
  formatted = ((time % 12) || 12) + pmOrAm
  option '', {value: time, selected: selected}, formatted.titleize()
```
Basically, it's for treating numbers in constants.schedule_time_options (not none, morning, evening) differently by not formatting them with pm/am.

Reply:
I think a regex would be safer. Right now, if someone adds an hour to the `schedule_time` num field but not `ScheduleRules.hours`, it will not be formatted correctly.

Reply:
Good point. It's more easily configurable if a regex is used. I've replaced the statement with `if time =~ /\A\d{1,2}\z/`.